docs/token-metadata/specification.md

```diff
@@ -33,7 +33,7 @@ pub struct Metadata {
 }
 ```
 
-The URI filed contains a pointer to the rest of the metadata. This metadata is stored off chain in most cases and the URI is usually a `https:` link to some decentralized storage.
+The URI field contains a pointer to the rest of the metadata. This metadata is stored off chain in most cases and the URI is usually a `https:` link to some decentralized storage.
```

Comment:
This file no longer exists. Could you please update your branch from `main` so the conflicts are resolved?